src/core/utils/arrayutils.js

```diff
@@ -1,9 +1,13 @@
 
 /**
- * Groups an array by a given key function.
- * @param {string} url
- * @param {params} Object
- * @returns {string}
+ * Groups an array into an object using a given key and value function, and an initial object
+ * to add to. By default the key and value functions will perform not perform any transformations
```

Comment:
key and value functions will not perform

Reply:
ugh sorry